test/unit/utils.py

```diff
@@ -14,37 +12,22 @@
 from pyquil.parser import parse
 from pyquil.quantum_processor import AbstractQuantumProcessor
 
+# Valid, sample Z85-encoded keys specified by zmq curve for testing:
+#   http://api.zeromq.org/master:zmq-curve#toc4
+CLIENT_PUBLIC_KEY = "Yne@$w-vo<fVvi]a<NY6T1ed:M$fCG*[IaLV{hID"
+CLIENT_SECRET_KEY = "D:)Q[IlAW!ahhC2ac:9*A}h:p?([4%wOTJ%JR%cs"
+SERVER_PUBLIC_KEY = "rq:rM>}U?@Lns47E1%kR.o@n%FcmmsL/@{H8]yf7"
+SERVER_SECRET_KEY = "JTKVSB%%)wK0E.X)V>+}o?pNmC{O&4W4b!Ni{Lh6"
+
 
 @contextmanager
 def run_rpcq_server(server: rpcq.Server, port: int):
     def run_server():
         server.run(endpoint=f"tcp://*:{port}", loop=asyncio.new_event_loop())
 
-    def check_server():
```

Comment:
This wasn't really providing any debug value. Dynamic port selection has been working better to address rpcq test issues.